src/workers/personalizedDigestEmail.ts

```diff
@@ -141,6 +144,24 @@ const digestTypeToFunctionMap: Record<
       },
     );
   },
+  [UserPersonalizedDigestType.ReadingStreakReminder]: async (data, con) => {
+    const { personalizedDigest, emailSendTimestamp, deduplicate = true } = data;
+    const emailSendDate = new Date(emailSendTimestamp);
```

Comment:
Naming wise it might become confusing as it's not email (I know the above also has it)

Reply:
Haha, yeah. I thought about doing some renaming of variables that say email in them 🤣🙈

Reply:
I guess also the cron runs hourly? I think so probably ok to not remove on read for now?

Reply:
Removed email from the variable name to make it less confusin